wiki/Article_styling_criteria/Formatting/en.md

```diff
@@ -613,19 +613,23 @@ There are two types of image links: inline and reference. Examples:
 **Inline style:**
 
 ```markdown
-![](/wiki/shared/flag/AU.gif)
+![Australian flag](/wiki/shared/flag/AU.gif "Australia")
 ```
 
 **Reference style:**
 
 ```markdown
-![][flag_AU]
+![Australian flag][flag_AU]
 
 [flag_AU]: /wiki/shared/flag/AU.gif "Australia"
 ```
 
 Images should use the inline linking style. References to reference links must be placed at the bottom of the article.
 
+The (first pair of) square brackets should contain a description of the image. It will be used by screen readers or when the image fails to load.
+
+The text between the quotation marks will be displayed below the image and when hovering the mouse cursor over the image.
```

Comment:
not sure if it matters but this feels a little repetitive with the `Alt text` and `Captions` sections. maybe this should stick to explaining the formatting only, and suggested usage goes in the other sections? maybe combine all the sections into this one? maybe it's fine as-is?

I'll just wait for another review...

Reply:
imo merging all three sections into a single one makes more sense. it would also allow for a shorter explanation, such as:

```markdown
Images are blah blah and have the following format:

![Alt text](/wiki/path/to/image.png "Title text")

Where:

- `Alt text` is ... . It is optional and does ... and ... .
- `Title text` is ... . It is optional and does ... and ... . It will be rendered as the image's caption if ... and ... .
```

Reply:
didn't do exactly this but I think I made it read well from top to bottom now